CloudCinemaBack/lib/cloud_cinema_back-stack.ts

```diff
@@ -0,0 +1,57 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as dynamodb from 'aws-cdk-lib/aws-dynamodb';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import path = require('path');
+import * as apigateway from 'aws-cdk-lib/aws-apigateway';
+
+
+
+
+// import * as sqs from 'aws-cdk-lib/aws-sqs';
+
+export class CloudCinemaBackStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const bucket = new s3.Bucket(this, 'CloudCinemaMoviesBucket', {
+      bucketName: 'cloud-cinema-movies-bucket', 
+      versioned: true, 
+      removalPolicy: cdk.RemovalPolicy.DESTROY
```

Comment:
Add this so we can delete a non-empty bucket when running `cdk destroy`
```suggestion
      removalPolicy: cdk.RemovalPolicy.DESTROY
      autoDeleteObjects: true
```